packages/curiefense/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,15 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -2,4 +2,10 @@
+ appVersion: 1.0.0
+ description: Complete curiefense deployment
+ name: curiefense
+-version: 1.0.0
+\ No newline at end of file
++version: 1.0.0
++
++kubeVersion: ">= 1.19"
++annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/release-name: curiefense
++  catalog.cattle.io/display-name: curiefense
```

Comment:
This specific annotation defines the name the UI should display and should be capitalized.
`catalog.cattle.io/display-name: Curiefense`

Reply:
We have to retake this. Let us see this weekend.

Reply:
@sergiopsyalo @JuanPabloGarciaMonzon any updates?